test/e2e/knit_irqwatch_test.go

```diff
@@ -0,0 +1,227 @@
+package e2eknit
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"math/rand"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"sort"
+	"time"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+	"github.com/openshift-kni/debug-tools/pkg/irqs"
+	"k8s.io/kubernetes/pkg/kubelet/cm/cpuset"
+)
+
+var _ = g.Describe("knit IRQ watch tests", func() {
+
+	var fixtureName = "xeon-multinuma-00"
+	var snapshotRoot string
+
+	type cmdOutput struct {
+		out []byte
+		err error
+	}
+
+	g.Context("Without isolated CPUs", func() {
+		g.It("Produces the expected delta between consecutive irq samples", func() {
+			cmdline := []string{
+				filepath.Join(binariesPath, "knit"),
+				"-P", filepath.Join(snapshotRoot, "proc"),
+				"-S", filepath.Join(snapshotRoot, "sys"),
+				"-W", "3s",
+				"-T", "1",
+				"-J",
+				"irqwatch",
+			}
+			fmt.Fprintf(g.GinkgoWriter, "running: %v\n", cmdline)
+
+			cmd := exec.Command(cmdline[0], cmdline[1:]...)
+			cmd.Stderr = g.GinkgoWriter
+
+			c := make(chan cmdOutput)
```

Comment:
isn't `cmd.Output()` running on a different goroutine?

Reply:
It does, but I provide the channel ```cmdOutput``` as an [argument](https://github.com/openshift-kni/debug-tools/pull/26/files#diff-105a28bc7bb59f3ba60810661320b99cfa11962075cc2117b2f504af0cf736ebR53) to the function that goroutine runs